OmniKitTests/PodStateTests.swift

```diff
@@ -13,7 +13,7 @@ class PodStateTests: XCTestCase {
     
     func testNonceValues() {
         
-        var podState = PodState(address: 0x1f000000, activatedAt: Date(), timeZone: .currentFixed, piVersion: "1.1.0", pmVersion: "1.1.0", lot: 42560, tid: 661771)
+        var podState = PodState(address: 0x1f000000, activatedAt: Date(), expiresAt: Date().addingTimeInterval(podSoftExpirationTime), timeZone: .currentFixed, piVersion: "1.1.0", pmVersion: "1.1.0", lot: 42560, tid: 661771)
```

Comment:
It doesn't really matter in this context, but might be good to fix nonetheless. `expiresAt` should be calculated using the same date instance that `activatedAt` uses, otherwise they are different times.

Reply:
and there are two more below